src/profile.c

```diff
@@ -338,7 +335,6 @@ static inline void outputStat(Stat stat, int exec, int visited)
   // Statement not attached to a file
   if(nameid == 0)
   {
-    HashUnlock(&profileState);
     return;
   }
 
```

Comment:
I think you also need to remove the Unlock at the end of this function (397)

Reply:
Damn, of course :/